protocol/test/forge/TempleTeamPaymentsFactory.t.sol

```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.4;
+
+import "forge-std/Test.sol";
+import "../../contracts/admin/TempleTeamPaymentsFactory.sol";
+
+contract TempleTeamPaymentsFactoryTest is Test {
+    //
+    uint256 testAllocationLength = 3;
+    uint256 testLastEpoch = 17;
+    address testUser = 0xa6E83976204969B278CcB5246EdB698Af6a0FAeF;
+
+    //
+    TempleTeamPaymentsFactory public factory;
+    IERC20 public temple = IERC20(0x470EBf5f030Ed85Fc1ed4C2d36B9DD02e77CF1b7);
+    address multisig = 0xe2Bb722DA825eBfFa1E368De244bdF08ed68B5c4;
+
+    function setUp() public {
+        factory = new TempleTeamPaymentsFactory(testLastEpoch);
+        factory.transferOwnership(multisig);
+    }
+
+    function testDirectPayouts() public {
+        uint256 prev = temple.balanceOf(testUser);
+
+        address[] memory recip = new address[](testAllocationLength);
+        for (uint256 i; i < recip.length; i++) {
+            recip[i] = testUser;
+        }
+
+        uint256[] memory values = new uint256[](testAllocationLength);
+        for (uint256 i; i < values.length; i++) {
+            values[i] = (i + 1) * 1 ether;
+        }
+        vm.startPrank(multisig);
+        temple.approve(address(factory), type(uint256).max);
+        factory.directPayouts(temple, recip, values);
+        vm.stopPrank();
+        assertGt(temple.balanceOf(testUser), prev);
```

Comment:
pls test for exact value user has to receive with `asertEq(currentBalance, prev + allocation)`.